Microsoft.TeamServices.Samples.Client/Release/ReleasesSample.cs

```diff
@@ -0,0 +1,512 @@
+            ReleaseHttpClient2 releaseClient = connection.GetClient<ReleaseHttpClient2>();
+
+            var releases = releaseClient.GetReleasesAsync(project: projectName).Result;
+
+            int releaseIdFilter = releases.FirstOrDefault().Id;
+
+            List<ReleaseApproval> releaseApprovals = new List<ReleaseApproval>();
+
+            // Iterate (as needed) to get the full set of approvals
+            int continuationToken = 0;
+            bool parseResult;
+            do
+            {
+                IPagedCollection<ReleaseApproval> releaseApprovalsPage = releaseClient.GetApprovalsAsync2(project: projectName, releaseIdsFilter: new List<int> { releaseIdFilter }, continuationToken: continuationToken).Result;
+
+                releaseApprovals.AddRange(releaseApprovalsPage);
+
+                int parsedContinuationToken = 0;
+                parseResult = int.TryParse(releaseApprovalsPage.ContinuationToken, out parsedContinuationToken);
+                if (parseResult)
+                {
+                    continuationToken = parsedContinuationToken;
+                }
+            } while ((continuationToken != 0) && parseResult);
+
+            // Show the approvals
+            foreach (ReleaseApproval releaseApproval in releaseApprovals)
+            {
+                Console.WriteLine("{0} {1}", releaseApproval.Id.ToString().PadLeft(6), releaseApproval.Status);
+            }
+
+            return releaseApprovals;
+        }
+
+        [ClientSampleMethod]
+
+        public void UpdateStatusOfAnApprovalFromPendingToApproved()
+        {
+            string projectName = ClientSampleHelpers.FindAnyProject(this.Context).Name;
+            string assignedToFilter = ClientSampleHelpers.GetCurrentUserName(this.Context);
+
+            // Get a release client instance
+            VssConnection connection = Context.Connection;
+            ReleaseHttpClient releaseClient = connection.GetClient<ReleaseHttpClient>();
+            ReleaseApproval updateApproval = new ReleaseApproval { Status = ApprovalStatus.Approved, Comments = "Good to go!" };
+
+            // Get all pending approval to the current user
+            IList<ReleaseApproval> releaseApprovalsPage = releaseClient.GetApprovalsAsync(project: projectName, assignedToFilter: assignedToFilter).Result;
+            ReleaseApproval releaseApprovalToApprove = releaseApprovalsPage.FirstOrDefault();
+
+            if (releaseApprovalToApprove != null)
+            {
+                ReleaseApproval approval = releaseClient.UpdateReleaseApprovalAsync(project: projectName, approval: updateApproval, approvalId: releaseApprovalToApprove.Id).Result;
+
+                Console.WriteLine("{0} {1}", approval.Id.ToString().PadLeft(6), approval.Status);
+            }
+        }
+
+        private static VisualStudio.Services.ReleaseManagement.WebApi.Release CreateRelease(ReleaseHttpClient releaseClient, int releaseDefinitionId, string projectName)
```

Comment:
The method is used at three place and they are independent to each other so felt its OK to have this method.